test/api.js

```diff
@@ -1179,6 +1179,38 @@ describe('/v1', function() {
       });
     });
 
+    describe.only('?scope=openid', function() {
+
+      function decodeJWT(b64) {
+        var jwt = b64.split('.');
+        return {
+          header: JSON.parse(Buffer(jwt[0], 'base64').toString('utf-8')),
+          claims: JSON.parse(Buffer(jwt[1], 'base64').toString('utf-8'))
+        };
+      }
+
+      it('should return an id_token', function() {
+        return newToken({ scope: 'openid' }).then(function(res) {
+          assert.equal(res.statusCode, 200);
+          assert(res.result.id_token);
+          var jwt = decodeJWT(res.result.id_token);
+          var header = jwt.header;
+          var claims = jwt.claims;
+
+          assert.equal(header.alg, 'RS256');
+          assert.equal(header.kid, config.get('openid.key').kid);
+
+          assert.equal(claims.sub, USERID);
+          assert.equal(claims.aud, clientId);
+          assert.equal(claims.iss, config.get('openid.issuer'));
+          var now = Math.floor(Date.now() / 1000);
+          assert(claims.iat <= now);
+          assert(claims.exp > now);
```

Comment:
We should check that it also returned an `access_token`, since that will be needed for hitting the UserInfo endpoint